app/src/schemas/toolSchemas.ts

```diff
@@ -5,33 +5,39 @@ import { assertUnreachable } from "@phoenix/typeUtils";
 
 import { JSONLiteral, jsonLiteralSchema } from "./jsonLiteralSchema";
 
+const jsonSchemaPropertiesSchema = z
```

Comment:
Love it, I wonder if it would be good to have a small set of jest tests that makes it easier to grok what's valid and not valid more declaratively. Would help ensure there are no regressions on the schema.

Reply:
We are testing this in various places already related to playground as well as `phoenix/app/src/schemas/__tests__/toolSchemas.test.ts`